scripts/monitoring/evalai_interface.py

```diff
@@ -119,8 +132,10 @@ def get_challenge_by_pk(self, challenge_pk):
         response = self.make_request(url, "GET")
         return response
 
-    def get_challenges(self):
+    def get_challenges(self, next_page_url=None):
         url = URLS.get("get_challenges")
         url = self.return_url_per_environment(url)
+        if next_page_url:
+            url = next_page_url
```

Comment:
This is not needed. Add a while loop in the auto-cancel script just like other scripts.

Reply:
I will need to write the request function then here only as the url is passed with a param for the next page. The other places have there own request function written and doesn't really use the eval interface

Reply:
@AyushR1 No. Have you checked EvalAI Interface? There is a `make_request` function that you can use.